test/geocoding_test.rb

```diff
@@ -5,49 +5,54 @@
 
 module Mapbox
   class GeocodingTest < Test::Unit::TestCase
-    should "#geocode_forward" do
+
+    def setup
       Mapbox.access_token = ENV["MapboxAccessToken"]
+    end
+
+    def teardown
+      Mapbox.request_open_timeout = nil
+      Mapbox.request_timeout = nil
+    end
+
+    should "#geocode_forward" do
       result = Mapbox::Geocoder.geocode_forward("Chester, NJ")
       assert result
+      assert Mapbox.request_opts[:timeout] = 80
+      assert Mapbox.request_opts[:open_timeout] = 30
```

Comment:
Maybe extract these asserts to separate case without verifying the `result` as we are interested only if parameters set correctly? Like `should "set_default_timeouts"`

Reply:
👍 , addressed